src/main/java/com/kobylynskyi/graphql/codegen/model/graphql/GraphQLRequest.java

```diff
@@ -25,8 +25,33 @@ public GraphQLResponseProjection getResponseProjection() {
         return responseProjection;
     }
 
+    /**
+     * @deprecated Not intended for use and will be removed in the next version.
+     * Please use one of: {@link #toHttpJsonBody} or {@link #toHttpQueryString}
+     */
     @Override
+    @Deprecated
     public String toString() {
-        return GraphQLRequestSerializer.serialize(this);
+        return toHttpJsonBody();
+    }
+
+    /**
+     * Serializes GraphQL request to be used as HTTP JSON body
+     * according to https://graphql.org/learn/serving-over-http specifications
+     * 
+     * @return the serialized request
+     */
+    public String toHttpJsonBody() {
+        return GraphQLRequestSerializer.toHttpJsonBody(this);
+    }
+
+    /**
+     * Serializes GraphQL request to be used as HTTP query string 
+     * according to https://graphql.org/learn/serving-over-http specifications
+     * 
+     * @return the serialized request
+     */
+    public String toHttpQueryString() {
```

Comment:
@kobylynskyi @joffrey-bion Finally, are you ok with `toHttpJsonBody` and `toHttpQueryString` ? If we mention HTTP for the `toHttpJsonBody` method, maybe it will be more consistent to mention HTTP for the `toHttpQueryString` method too.

If you don't agree with that, I can rename the method and all mentions to it, no problem :)

Reply:
Hi Laura, sorry for the delay. I was actually envisioning `toQueryString()` to be the raw GraphQL query string, not the HTTP one, so that people can build upon it. Actually, `toHttpQueryString()` would be a good *addition* to that (which would URL-encode the query returned by `toQueryString()` I guess), but I believe we should keep the raw GraphQL query available to users for extension.
What do you guys think?

Reply:
@joffrey-bion I totally agree with you! Indeed, I didn't think about the raw GraphQL query string. It makes sense. So, I will rename it.

About an additionnal `toHttpQueryString()` or `toUrlQueryString()` (both name are fine to me too), it would be nice. I can add it but maybe in other issue, I have no more time for this week 😕 

Thanks Joffrey!